layouts/shortcodes/cncf-landscape.html

```diff
@@ -57,11 +57,27 @@
 </script>
 {{- end -}}
 <div id="frameHolder">
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/iframe-resizer/4.3.9/iframeResizer.min.js"></script>
   {{ if ( .Get "category" ) }}
-  <iframe frameborder="0" id="landscape" scrolling="no" src="https://landscape.cncf.io/card-mode?category={{ .Get "category" }}&grouping=category&embed=yes" style="width: 1px; min-width: 100%"></iframe>
+  <iframe id="iframe-landscape" src="https://landscape.cncf.io/embed/embed.html?key={{ .Get "category" }}&headers=false&style=shadowed&size=md&bg-color=%23d95e00&fg-color=%23ffffff&iframe-resizer=true" style="width: 1px; min-width: 100%; min-height: 100px; border: 0;"></iframe>
+  <script>
+    iFrameResize({ }, '#iframe-landscape');
+  </script>
   {{ else }}
-  <iframe frameborder="0" id="landscape" scrolling="no" src="https://landscape.cncf.io/card-mode?embed=yes" style="width: 1px; min-width: 100%" title="CNCF Landscape"></iframe>
+  <style>
+    /* Display subcategory name under sticky header when using anchor */
+    #iframe-landscape-kcsp, #iframe-landscape-conformance, #iframe-landscape-ktp {
+      scroll-margin-top: 35px;
+    }
+  </style>
```

Comment:
We'd prefer to have this in the page header (use `.HasShortcode` to handle that).